README.md

```diff
@@ -32,7 +32,7 @@ $ tsd query jquery.*
 $ tsd query "jquery.*"
 
 #install all definitions from tsd.json
-$ tsd install
+$ tsd reinstall
```

Comment:
Hi @perkastman! The `install` command performs like the `reinstall` command in this case. You can update the doc informing that the two ways are correct.